docs/API/Users_Guide/Submission.md

```diff
@@ -2319,6 +2319,71 @@ curl --header "X-Auth-Token: $token" 'https://api.gdc.cancer.gov/v0/submission/T
 
 Submitters can use the GraphQL query language for advanced search and retrieval of data from the GDC Submission Portal. See [GraphQL](#querying-submitted-data-using-graphql) for more information.
 
+## Patching Entitites
+
+The GDC Submission API supports the HTTP PATCH method for updating existing entities with additional fields.
+
+**PATCH** can be used to add extra fields to an existing entity, without requiring the submission of required fields. 
+
+The PATCH method cannot be used to create new entities, and the provided submitter_id must match an existing submitter_id. 
+
```

Comment:
Here I would recommend adding another JSON that was the "original" one

Reply:
Done!